python/lib/dependabot/python/file_parser/setup_cfg_file_parser.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative "setup_file_parser_base"
+
+module Dependabot
+  module Python
+    class FileParser
+      class SetupCfgFileParser < SetupFileParserBase
```

Comment:
It seems a little unclear to me having a subclass for `setup.cfg` fiels that only overrides the `function` method. I think it would be easier to understand if this was all in `SetupFileParser` and it would check which function to use based on the `dependency_files`.

Reply:
Or actually, thinking about it more, maybe the native helper should parse both setup files for us, and then we only need ti call `SetupFileParser.new(..).dependency_set` once, and we'd get the results from both files? It seems like the results that we get from the native helpers are mostly identical, if `parse_setup` would append the results from `parse_setup_cfg`, I think it would all just work?

WDYT?

Reply:
Yeah that sounds very reasonable. I was planning to do that initially then decided not to touch existing parser code. It seems we all favour that option, so I will go ahead.

BTW in reality, I hope no one would ever split dependencies into `setup.cfg` and `setup.py`. I'm not even sure that would work. :D